### DISCUSSION TITLE
New NICs in Instance Not Updated After Reboot

### DISCUSSION THREAD

**Discussion by bds-congnguyen** [apache/cloudstack#9753] [General]
Hi everyone,

I encountered an issue after adding additional NICs to an instance. Each time the instance reboots, the new NICs remain down. I attempted running `cloud-init clean` followed by a reboot, but it seems the new interfaces are not being updated in `/etc/netplan/50-cloud-init.yaml`, causing them to stay down.

Is there a way to automate this process or avoid manually editing the /etc/netplan configuration file? Ideally, we want to manage everything through Cloudstack, without needing to touch system files directly.

Thanks in advance for your help!

**Comment by rajujith** [comment]
@bds-congnguyen  I am assuming that the interface remains plugged into the instance after reboot. If you have configured the netplan with two interfaces it should get configured right.

**Reply by bds-congnguyen** [reply]
yeah, but the 1st NIC Netplan configuration was generated by cloud-init. if we add another NICs into instance then it's should be setup automatically instead of add a Netplan manually. What do you think?

**Reply by weizhouapache** [reply]
can you share the netplan file ?
@bds-congnguyen

**Comment by DaanHoogland** [comment]
@bds-congnguyen , assuming this is no longer an issue. Please comment if you think it still is.
